SwiftGit2/Repository.swift

```diff
@@ -376,12 +376,12 @@ final public class Repository {
 	/// If the reference is a branch, a `Branch` will be returned. If the
 	/// reference is a tag, a `TagReference` will be returned. Otherwise, a
 	/// `Reference` will be returned.
-	public func referenceWithName(_ name: String) -> Result<ReferenceType, NSError> {
+	public func reference(withName name: String) -> Result<ReferenceType, NSError> {
```

Comment:
Changed to `reference(named:)` as per https://github.com/SwiftGit2/SwiftGit2/pull/72#discussion_r93708682